db-manager/project_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+pipenv install --dev --three
+# enable once tests have been added
+# pipenv run python -m pytest
```

Comment:
I might need a second pair of eyes. I commented that line out but it still seem to run pytest on this project?

Reply:
We are not passing the `IMAGE_TAG` correctly so it may be running on older images.